test/test.js

```diff
@@ -33,20 +32,48 @@ describe('StaticI18n', function(){
 
 describe('Translator', function(){
 
-  it('consumes translation catalogs', function() {
-    var translator = new Translator({localeDir: appPath + '/locale'});
+  var baseDir = path.join(__dirname, '/fixtures/app');
+  var singleOptions = {localeDirs: [baseDir + '/locale']};
+  var multiOptions = {
+    localeDirs: [baseDir + '/locale', baseDir + '/installed_deps/locale']
+  };
+  var hello = 'Hello World';
+  var bonjour = 'Bonjour tout le monde';
+
+  var testGetLocales = function(options) {
+    var translator = new Translator(options);
     var locales = translator.getLocales();
     expect(locales.length).to.equal(3);
+  };
+
+  var testFrTranslate = function(options, string, translated) {
+    var translator = new Translator(options);
+    expect(translator.langGettext('fr', string)).to.equal(translated);
+  };
+
+  it('consumes translation catalogs', function() {
+    testGetLocales(singleOptions);
```

Comment:
Keeping the assertion in a helper function makes it difficult to read what the test is actually doing.

If you look at our tests, you'll often see duplication because we don't want to generalize.  Each test should be their own discrete bit of work.

Rewrite these guys without the helper functions.  It looks like you're trying to test using multiple locales, it may be best to introduce a new describe block:

```
describe('Translator', function(){
  describe('configured with multiple locale dirs', function() {
    it('merges catalogs that belong to the same language', function() {
      ...
```

If you need help with this, let me know.